internal/server/errors.go

```diff
@@ -35,36 +37,45 @@ func sendAPIError(c *gin.Context, err error) {
 		resp.Code = http.StatusUnauthorized
 		// hide the error text, it may contain sensitive information
 		resp.Message = "unauthorized"
+
 	case errors.Is(err, internal.ErrForbidden):
 		resp.Code = http.StatusForbidden
 		// hide the error text, it may contain sensitive information
 		resp.Message = "forbidden"
-	case errors.Is(err, internal.ErrDuplicate):
+
+	case errors.As(err, &uniqueConstraintError):
```

Comment:
Should we also populate the `FieldErrors`, since the constraint is likely related to a field?  We'll have to make sure we can easily translate the column name to a field name.  If that's a challenge it should be fine to leave that part for a future PR.

Reply:
Field errors should probably be omitted for now. If we need to expose the field in the future, we can add it then

Reply:
Sounds good. I guess we'll see what the CLI needs from the API, and we can also add it at that time if we need it